src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -303,10 +303,11 @@ const getAuthenticatorButtonElements = (
   return reorderAuthenticatorButtons(formattedOptions, deviceKnown);
 };
 
-export const getOVMethodTypeAuthenticatorButtonElements = (
+export const getAppAuthenticatorMethodButtonElements = (
   authenticator: Input,
   step: string,
   deviceKnown?: boolean,
+  authKey = AUTHENTICATOR_KEY.OV,
```

Comment:
Does lint give an error since authKey is placed after `deviceKnown`? Maybe not since authKey has a default value, is there any issue swapping `deviceKnown` and authKey?

Reply:
yeah looks like lint didn't give an error since authKey has a default value.  No real issue with swapping them, theres just a couple places where I had to swap arguments to match, no biggie.